ro_help/hub/views.py

```diff
@@ -53,16 +52,15 @@ def get_context_data(self, **kwargs):
         if not ngo:
             return context
 
-        page = self.request.GET.get("page")
-
         needs = ngo.needs.filter(resolved_on=None)
         if "need" in kwargs:
             needs = needs.exclude(pk=kwargs["need"].pk)
             context["current_need"] = kwargs["need"]
 
         for kind in KIND.to_list():
             kind_needs = needs.filter(kind=kind)
-            needs_paginator = paginator.Paginator(kind_needs, self.paginated_by)
+            needs_paginator = paginator.Paginator(kind_needs, 3)
```

Comment:
Let's keep `paginated_by ` as a class attribute. Maybe also we can move it as a constant.